odoo/tools/mail.py

```diff
@@ -257,6 +257,8 @@ def html_sanitize(src, silent=True, sanitize_tags=True, sanitize_attributes=Fals
 
 URL_REGEX = r'(\bhref=[\'"](?!mailto:|tel:|sms:)([^\'"]+)[\'"])'
 TEXT_URL_REGEX = r'https?://[a-zA-Z0-9@:%._\+~#=/-]+(?:\?\S+)?'
+# allow the retrieve the text content of the HTML elements
```

Comment:
Not sure if there is a word missing or too much in this sentence but I don't understand this comment ;-)
Anyway, are you sure about your groups? [regexpr](https://regexper.com/#%28%5Cbhref%3D%5B%5C'%22%5D%28%3F!mailto%3A%7Ctel%3A%7Csms%3A%29%28%5B%5E%5C'%22%5D%2B%29%5B%5C'%22%5D%29%28%5B%5E%3C%3E%5D*%3E%28%5B%5E%3C%3E%5D%2B%29%3C%5C%2F%29%3F)

Reply:
@mart-e yes, pretty sure, it work perfectly for me
It's used to get the content of the `<a>`

e.g.
`<a href="url" title="...">bouh</a>`
We want `url` and `bouh` :)

Reply:
Oh right, missed the `?` for the third group. Anyway "retrieve inner content of the link" or something like this for the comment

Reply:
ok :)